wicket-orientdb/src/main/java/ru/ydn/wicket/wicketorientdb/utils/query/filter/CollectionFilterCriteria.java

```diff
@@ -0,0 +1,45 @@
+package ru.ydn.wicket.wicketorientdb.utils.query.filter;
+
+import com.google.common.base.Strings;
+import org.apache.wicket.model.IModel;
+import ru.ydn.wicket.wicketorientdb.utils.query.filter.value.IFilterValue;
+import ru.ydn.wicket.wicketorientdb.utils.query.filter.value.CollectionFilterValue;
+
+import java.util.List;
+
+/**
+ * Collection filter
+ * SELECT FROM Class WHERE num IN ['1', '2', '3']
+ */
+public class CollectionFilterCriteria extends AbstractFilterCriteria {
+
+    private final IFilterValue filterValue;
+
+    public <T> CollectionFilterCriteria(String field, CollectionFilterValue<T> filterValue, IModel<Boolean> join) {
+        super(field, FilterCriteriaType.LIST + "_" + field, join);
+        this.filterValue = filterValue;
+    }
+
+
+    @Override
+    protected String apply(String field) {
```

Comment:
Disagree with special construction of sql per criteria for multiple values. Instead of listing all in SQL you can use something like `select from MyClass where link in :mylist` and than use IModel<List<?>> for `mylist`